### ISSUE TITLE
Sites contrast borders on violating accessibility guidelines

### ISSUE THREAD

**Issue by felixgao-0** [hackclub/nest#58]
See title thats it basically, see example below.

- The purple in `$ ssh hackclub.app` and more places

![image](https://github.com/user-attachments/assets/db20f3da-0f94-4753-8af6-425caa4438ee)

From my limited search, the purple above appears to be the only colour not contrasty enough (is that a word?). 
However, it may be worth increasing the contrast of other colours used, like the gray buttons below (oh and another appearance of that purple).

![image](https://github.com/user-attachments/assets/4b5b0425-79ba-4094-bda1-d40b6c845bc0)

Not rather sure if this is something which needs to be changed, or if I'm just overly worried, but thought I'd just make an issue. Let me know if I'm just yapping for no reason lol.

**Comment by polypixeldev** [comment]
Good catch! I'll PR a fix to those colors later today
